tests/test.js

```diff
@@ -201,6 +213,18 @@ describe('Autographa Test', () => {
       .getText('#chapterBtn').should.eventually.equal('1');
     });
 
+    it('should check verse existance after 3x, 4x click', () => {
```

Comment:
Test for this after 'should change layout to 2x'

Reply:
We cannot put after 2x because, when we click 3x and 4x it simultaneously checks 
**'should check highlight verse on 3x'
'should check highlight verse on 4x'**
and if we but after 2x layout it fires both the above cases as an error since it cannot check highlights without clicking the button.
So I added **should check verse existance after 3x, 4x click** before **'should check highlight verse on 3x'**

Reply:
Could explain this again?